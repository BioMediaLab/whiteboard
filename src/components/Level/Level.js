import React from 'react'
import './Level.css'

export default function Level({ children }) {
  return <div className="Level">{children}</div>
}

export function LevelItem({ children }) {
  return <div className="LevelItem">{children}</div>
}
