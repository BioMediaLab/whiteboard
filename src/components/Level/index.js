export { default, LevelItem } from './Level'
