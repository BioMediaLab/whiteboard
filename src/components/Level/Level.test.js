import React from 'react'
import Level from './'

describe('Level', () => {
  test('can render', () => {
    const component = render(
      <Level>
        {children}
      </Level>
    )
  
    expect(component).toMatchSnapshot()
  })
})

