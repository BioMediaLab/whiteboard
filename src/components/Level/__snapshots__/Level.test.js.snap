// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level can render 1`] = `
<div>
  <div
    class="Level"
  >
    <div
      class="LevelItem"
    >
      first
    </div>
    <div
      class="LevelItem"
    >
      second
    </div>
    <div
      class="LevelItem"
    >
      third
    </div>
  </div>
</div>
`;
